recipe/meta.yaml

```diff
@@ -1,14 +1,23 @@
-{% set version = "6.1.6" %}
+{% set version = "7.0.0" %}
+{% set python_min = python_min | default("3.11") %}
+{% set PYTHON = PTYHON | default("$PYTHON") %}
```

Comment:
It seems that `$PYTHON` isn't defined on windows?

Reply:
Gross. Have also hit some stuff like this on `uv`. Not sure how it works in this situation... perhaps moving that behavior inside the `if` for checking if the `noarch` package is going to be built? Alternately:
```
{% set PYTHON = PTYHON | default("$PYTHON") %}
{% set PYTHON = PTYHON | default("%PYTHON%") %}  # [win]
```

Alternately, if there's only one `noarch`, maybe it should be `noarch: python`, which would then use the [CFEP-25](https://github.com/conda-forge/cfep/blob/main/cfep-25.md) `python_min` in more places:

```
build:
  noarch: python
requirements:
  host:
    - python {{ python_min }}
  run:
    - python >= {{ python_min }} 
test:
  requires:
    - python {{ python_min }}
```

Reply:
On second thoughts, that fills me with dread :laughing:

Reply:
Arghl it dosent work.

Reply:
I changed to noarch: python it seems to work